dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -0,0 +1,291 @@
+    [[ "$ws_create" == *"STOPPED"* ]] || exit 1
+    ws_id=$(echo "${ws_create}" | $JQ_BINARY -r '.id')
+    [[ "$ws_id" == *"workspace"* ]] || exit 1
+    echo "[TEST] workspace '$ws_name' created succesfully"
+
+    # start workspace
+    curl -s "http://che-${OPENSHIFT_NAMESPACE_URL}/api/workspace/${ws_id}/runtime?environment=default" \
+    -H 'Content-Type: application/json;charset=UTF-8' \
+    -H 'Accept: application/json, text/plain, */*' \
+    -H 'Connection: keep-alive' \
+    --data-binary '{}' \
+    --compressed
+   wait_workspace_status "RUNNING"
+   echo "[TEST] workspace '$ws_name'started succesfully"
+   #TODO maybe add more checks that state is good
+
+   # stop workspace
+   ws_stop=$(curl -s "http://che-${OPENSHIFT_NAMESPACE_URL}/api/workspace/${ws_id}/runtime?create-snapshot=false" -X DELETE \
+   -H 'Accept: application/json, text/plain, */*' \
+   -H 'Connection: keep-alive' \
+   --compressed \
+   -o /dev/null \
+   --write-out '%{http_code}')
+   [[ "$ws_stop" = "204" ]] || exit 1
+   wait_workspace_status "STOPPED"
+   echo "[TEST] workspace '$ws_name' stopped succesfully"
+
+   # remove workspace
+   ws_remove=$(curl -s "http://che-${OPENSHIFT_NAMESPACE_URL}/api/workspace/${ws_id}" -X DELETE \
+   -H 'Accept: application/json, text/plain, */*' \
+   -H 'Connection: keep-alive' \
+   --compressed \
+   -o /dev/null \
+   --write-out '%{http_code}')
+   [[ "$ws_remove" = "204" ]] || exit 1
+   check_ws_removed=$(curl -s "http://che-${OPENSHIFT_NAMESPACE_URL}/api/workspace" \
+   -H 'Accept: application/json, text/plain, */*' \
+   -H 'Connection: keep-alive' \
+   --compressed)
+   [[ "$check_ws_removed" != *"${ws_id}"* ]] || exit 1
+   echo "[TEST] workspace '$ws_name' removed succesfully"
+}
+
+destroy_ocp() {
+    $OC_BINARY login -u system:admin
+    $OC_BINARY delete pvc --all
+    $OC_BINARY delete all --all
+    $OC_BINARY cluster down
+}
+
+parse_args() {
+    HELP="valid args: \\n
+    --run-ocp - run ocp cluster\\n
+    --destroy - destroy ocp cluster \\n
+    --deploy-che - deploy che to ocp \\n
+    --multiuser - deploy che in multiuser mode \\n
+    --test -  run simple test which will create > start > stop > remove CHE workspace\\n
+    =================================== \\n
+    ENV vars \\n
+    CHE_IMAGE_TAG - set CHE images tag, default: nightly \\n
+    CHE_MULTIUSER - set CHE multi user mode, default: false (single user) \\n
+"
+
+
+
+    if [ $# -eq 0 ]; then
+        echo "No arguments supplied"
+        echo -e "$HELP"
+        exit 1
+    fi
+
+    if [[ "$@" == *"--multiuser"* ]]; then
+      CHE_MULTIUSER=true
+    fi
+
+    init
+
+    for i in "${@}"
+    do
+        case $i in
+           --run-ocp)
+               run_ocp
+               shift
+           ;;
+           --destroy)
+               destroy_ocp
+               shift
+           ;;
+           --deploy-che)
+               deploy_che_to_ocp
+               shift
+           ;;
+           --test)
```

Comment:
please, consider adding failure on detection of unexpected option

Reply:
done